isis/tests/BundleControlPointTests.cpp

```diff
@@ -0,0 +1,277 @@
+#include "Angle.h"
```

Comment:
I think that this file should be named UnitTestBundleControlPoint.cpp according to the convention set by the gtest conversion project (https://github.com/USGS-Astrogeology/ISIS3/wiki/Writing-ISIS3-Tests-Using-Gtest-and-Ctest#test-names).